src/RoverSelection/RoverSelection.tsx

```diff
@@ -1,6 +1,124 @@
+              <th> 7 </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734;</th>
+            </tr>
+            <tr>
+              <th>Weight</th>
+              <th> 899kg </th>
+              <th> &#9733; &#9734; &#9734; &#9734; &#9734;</th>
+            </tr>
+          </table>
+        </div>
+        <div className='OpportunityRover'>
+          <h2> Opportunity Rover:</h2>
+          <img src='https://upload.wikimedia.org/wikipedia/commons/thumb/d/d8/NASA_Mars_Rover.jpg/1200px-NASA_Mars_Rover.jpg' width='200px' height='200px' alt='Image of Opportunity Rover' />
+          <h3> Stats</h3>
+          <table className='OpportunityTable'>
+            <tr>
+              <th>Date Landed</th>
+              <th>07/07/2003</th>
+              
+            </tr>
+            <tr>
+              <th>Speed</th>
+              <th> 0.1mph </th>
+              <th> &#9733; &#9733; &#9733; &#9734; &#9734; </th>
+            </tr>
+            <tr>
+              <th>Size</th>
+              <th> 190 cubic feet </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734; </th>
+            </tr>
+            <tr>
+              <th>Life Span</th>
+              <th> 785 weeks </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9733;</th>
+            </tr>
+            <tr>
+              <th>Number of Cameras</th>
+              <th> 5 </th>
+              <th>&#9733; &#9733; &#9733; &#9734; &#9734;</th>
+            </tr>
+            <tr>
+              <th>Weight</th>
+              <th> 180kg </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734;</th>
+            </tr>
+          </table>
+        </div>
+
+        <div className='SpiritRover'> 
+          <h2> Spirit Rover:</h2>
+          <img src='https://brownspaceman.com/wp-content/uploads/2014/01/spirit-rover.jpg' alt='Image of Spirit Rover' width='200px' height='200px'/>
+          <h3> Stats</h3>
+          <table className='SpiritTable'>
+            <tr>
+              <th>Launch Date</th>
+              <th>10/06/2003 </th>
+             
+            </tr>
+            <tr>
+              <th>Speed</th>
+              <th> 0.1mph </th>
+              <th> &#9733; &#9733; &#9733; &#9734; &#9734; </th>
+            </tr>
+            <tr>
+              <th>Size</th>
+              <th> 190 cubic feet </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734;</th>
+            </tr>
+            <tr>
+              <th>Life Span</th>
+              <th> 324 weeks </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734;</th>
+            </tr>
+            <tr>
+              <th>Number of Cameras</th>
+              <th> 5 </th>
+              <th>&#9733; &#9733; &#9733; &#9734; &#9734;</th>
+            </tr>
+            <tr>
+              <th>Weight</th>
+              <th> 180kg </th>
+              <th>&#9733; &#9733; &#9733; &#9733; &#9734;</th>
+            </tr>
+          </table>
+        </div>
+
```

Comment:
we can get rid of these empty new lines :)

Reply:
done :)